### DISCUSSION TITLE
Select files by extension?

### DISCUSSION THREAD

**Discussion by KreepyKelly** [qarmin/czkawka#925] [General]
Is there a way to select just the mp3 or flac files when using Music Duplicates? I tried to do a Custom Select, but my attempts failed. I tried searching the discussions, but am guessing it's common knowledge that I don't know.
Thank you for your help!

**Comment by frasu** [comment]
Hi, I've just checked it and yes, it's possible.

In Select Custom you need to check the "Regex Path + Name" and type
```
.*mp3
```
Then all your mp3 files will be selected.
